### ISSUE TITLE
Evaluation score is 43.0% mAP

### ISSUE THREAD

**Issue by DevKiHyun** [TaoRuijie/TalkNet_ASD#44]
Hi.

First, thank you for publish your source code.

I ran your code to check it is working.

I had some troubles when download and preprocess datasets (some files are cracked during downloading). But, after re-download, the problem is gone. 

After preprocessing, I ran evaluation code with your pre-trained model, but I got 43.09% score for val set. And for test set, I got 100% (But, I found a issue page for test set score, so it is okay).

So, What problem happen to me?
Is the dataset can corrupted? However, I randomly check video files, there seems to be no problem with video and audio.
Or library version issue? I set up with your requirement file.
I can't guess any reason of this problem.

Can you guess this situation?

Thank you for reading. And I apologize for my poor English.

**Comment by TaoRuijie** [comment]
are you using the pre-trained model for AVA? Not pretrain_TalkSet.model, ought to be pretrain_AVA.model

**Comment by DevKiHyun** [comment]
Holy... Thank you!. By my mistake, download link of code is changed with TalkSet.model link. 
Now, I got correct mAP score!
